Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -1,11 +1,6 @@
 #if !os(Linux)
 import XCTest
 
-/// Enhances failure messages with a command line diff tool expression that can be copied and pasted into a terminal.
-///
-///     diffTool = "ksdiff"
-public var diffTool: String? = nil
```

Comment:
Any idea on how to restore this bit of global configuration? The `Format` generic makes it tricky, but maybe we could use some kind of lookup table...

``` swift
SnapshotTesting.inform[Data.self] = .basic
SnapshotTesting.inform[String.self] = .basic
SnapshotTesting.inform[UIImage.self] = .basic
```

Alternately we could drop the generic for now till we have a compelling reason to introduce it...

Reply:
I think we should not delete the `diffTool` var and keep backwards compatibility.

The `.basic` Informing would do the same as the current behavior:
```
diffTool
	        .map { "\($0) \"\(snapshotFileUrl.path)\" \"\(failedSnapshotFileUrl.path)\"" }
	        ?? "@\(minus)\n\"\(snapshotFileUrl.path)\"\n@\(plus)\n\"\(failedSnapshotFileUrl.path)\""
```
Informing would default to `.basic` and the behavior would be unchanged.


Regarding the global configuration, a lookup table should work. I might be focusing too much on the use case I have I mind, but it might be useful to be able to configure different diff tools for different Formats.

Let me know when you figure out the next steps 👍

Reply:
@stephencelis Any update? :D